app/Services/PostService.php

```diff
@@ -65,7 +65,7 @@ public function update($signup, $data, $transactionId)
         // @TODO: This will is only temporary and will be removed!
         // If this is a signup update, get the most recent post.
         // If there is a quantity_pending, this is a signup.
-        if ($post->quantity_pending) {
+        if ($post->quantity || $post->quantity_pending) {
```

Comment:
@sbsmith86 that sounds reasonable! everything seems to be working so I don't think it's breaking anything, I just couldn't figure out what it was doing. It says in the comment that if there is `quantity_pending` then it's a signup, but that's not necessarily true so that confused me. But yeah seems like we can remove, I don't have a strong preference for if that's in this PR or not.